src/dcap_provider.cpp

```diff
@@ -169,6 +228,59 @@ static std::string get_base_url()
     return env_base_url;
 }
 
+static std::string get_base_url_tdx(bool requestingCertificate = false)
+{
+    std::string result;
+
+    std::string env_base_url_cert =
+        get_env_variable(ENV_AZDCAP_BASE_URL_TDX_CRL);

```

Comment:
Changed it so that it's no longer necessary to have two environment variables for TDX. Name is now AZDCAP_BASE_CERT_URL_TDX.

This is only reaching service for collateral, not for the certificate.